views/newEvent.handlebars

```diff
@@ -1,154 +1,112 @@
-<div class="container">
+        <div class="container d-flex justify-content-center">
+            <form>
 
+                <div class="row" style="margin-bottom: 30px;">
+                    <div class="twelve columns" style="text-align: center;">
+                    <h2>Create an Event</h2>
+                    <p>Your first steps to creating your event begin with these few questions. You'll be on your way to securing new volunteers in no time!</p>
+                    </div>
+                </div>
+            
+                <div class="form-row">
+                  <div class="form-group col-md-12">
+                    <label for="inputEvent">Event Name</label>
```

Comment:
I think the for="inputEvent" needs to match the id="inputEventName" on the line below (and it doesn't)

Reply:
Not sure those need to match but I'm double checking the code right now

Reply:
I think this is what ties the label to the input.